pr-data.csv

```diff
@@ -1068,7 +1068,7 @@ https://github.com/Alluxio/alluxio,68fcda9dd2ca2181eb897e10ca02fa5bc3a3099f,core
 https://github.com/Alluxio/alluxio,68fcda9dd2ca2181eb897e10ca02fa5bc3a3099f,core/server/worker,alluxio.worker.block.allocator.MaxFreeAllocatorTest.allocateBlock,ID,Deleted,,https://github.com/Alluxio/alluxio/pull/17638/commits/a1b04a80c52005a8eb12c3be37d24d38d621e27e
 https://github.com/Alluxio/alluxio,68fcda9dd2ca2181eb897e10ca02fa5bc3a3099f,core/server/worker,alluxio.worker.block.allocator.RoundRobinAllocatorTest.allocateBlock,ID,Deleted,,https://github.com/Alluxio/alluxio/pull/17638/commits/a1b04a80c52005a8eb12c3be37d24d38d621e27e
 https://github.com/Alluxio/alluxio,e6d76803f27133d7700811585f5310470e50e487,core/server/worker,alluxio.worker.block.BlockLockManagerTest.lockAlreadyReadLockedBlock,ID,DeveloperFixed,,https://github.com/Alluxio/alluxio/commit/921a053d0feafc3c831743d1ae996b31d3cc1288
-https://github.com/Alluxio/alluxio,e6d76803f27133d7700811585f5310470e50e487,core/server/worker,alluxio.worker.block.BlockLockManagerTest.lockAlreadyWriteLockedBlock,ID,,,
+https://github.com/Alluxio/alluxio,e6d76803f27133d7700811585f5310470e50e487,core/server/worker,alluxio.worker.block.BlockLockManagerTest.lockAlreadyWriteLockedBlock,ID,DeveloperFixed,,https://github.com/Alluxio/alluxio/commit/7e393ef92a26284e83907ad08e21ac70aeafdb83
```

Comment:
Please email me privately your VM ID and absolute path to the log files confirming that this commit fixed this test.